F/fakechroot/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "fakechroot"
+version = v"2.20.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/dex4er/fakechroot.git", "b42d1fb9538f680af2f31e864c555414ccba842a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/fakechroot
+./autogen.sh 
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```

Comment:
Try `--enable-shared` or something to get the shared library?

Reply:
Isn't it `lib/fakechroot`?

Reply:
It looks to me like it's only installing the static library, e.g. `libfakechroot.a`

Reply:
Nope, I'm totally wrong.  Yeah, we just need to update the search path for the `LibraryProduct`.

Reply:
I assume the wizard should catch this though.

Reply:
https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/745